pyvocz/app.py

```diff
@@ -59,6 +60,16 @@ def add_language_code(endpoint, values):
         if app.url_map.is_endpoint_expecting(endpoint, 'lang_code'):
             values['lang_code'] = g.lang_code
 
+    @app.before_request
+    def redirect_subdomains():
+        url = urlparse(request.url)
+        hostname = url.hostname
+        hostname_parts = hostname.split('.')
+        if len(hostname_parts) > 2:
+            subdomain = hostname_parts[0]
+            if subdomain == 'www':
+                return redirect(request.url.replace('www.', ''))
```

Comment:
Ještě jsem to limitoval na netloc: bd54619e716c259cdbdc70b8b6711a07ae44b743

Díky!